noir-projects/noir-contracts/contracts/account/simulated_account_contract/src/main.nr

```diff
@@ -0,0 +1,49 @@
+mod public_key_note;
+
+use dep::aztec::macros::aztec;
+
+#[aztec]
+pub contract SimulatedAccount {
```

Comment:
Please add here docs of what is this used for. I also don't think the name is descriptive enough. Maybe `SimulationMockAccount`?